packages/nest/src/schematics/application/application.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { Tree, VirtualTree } from '@angular-devkit/schematics';
+import { createEmptyWorkspace } from '@nrwl/schematics/testing';
+import { runSchematic } from '../../utils/testing';
+
+describe('app', () => {
+  let appTree: Tree;
+
+  beforeEach(() => {
+    appTree = new VirtualTree();
```

Comment:
Please use `Tree.empty()`

`VirtualTree` is going away in `@angular-devkit@8.0.0`